templates/types/extractor/fastapi/pyproject.toml

```diff
@@ -13,7 +13,10 @@ python = "^3.11,<4.0"
 fastapi = "^0.109.1"
 uvicorn = { extras = ["standard"], version = "^0.23.2" }
 python-dotenv = "^1.0.0"
-llama-index = "^0.10.58"
+llama-index = "^0.11.1"
+llama-index-llms-openai = "^0.2.0"
```

Comment:
we're not adding dependencies for extractor? how do we then support a different vector db that is selected?